src/main/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/recipe/AddRecipeCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package ay2122s1_cs2103t_w16_2.btbb.logic.parser.recipe;
+
+import static ay2122s1_cs2103t_w16_2.btbb.logic.parser.util.CliSyntax.PREFIX_RECIPE_INGREDIENT;
+import static ay2122s1_cs2103t_w16_2.btbb.logic.parser.util.CliSyntax.PREFIX_RECIPE_NAME;
+import static ay2122s1_cs2103t_w16_2.btbb.logic.parser.util.CliSyntax.PREFIX_RECIPE_PRICE;
+
+import java.util.ArrayList;
+
+import ay2122s1_cs2103t_w16_2.btbb.commons.core.Messages;
+import ay2122s1_cs2103t_w16_2.btbb.exception.ParseException;
+import ay2122s1_cs2103t_w16_2.btbb.logic.commands.recipe.AddRecipeCommand;
+import ay2122s1_cs2103t_w16_2.btbb.logic.descriptors.RecipeDescriptor;
+import ay2122s1_cs2103t_w16_2.btbb.logic.parser.Parser;
+import ay2122s1_cs2103t_w16_2.btbb.logic.parser.util.ArgumentMultimap;
+import ay2122s1_cs2103t_w16_2.btbb.logic.parser.util.ArgumentTokenizer;
+import ay2122s1_cs2103t_w16_2.btbb.logic.parser.util.ParserUtil;
+import ay2122s1_cs2103t_w16_2.btbb.model.order.RecipeIngredientList;
```

Comment:
Since we are using the same RecipeIngredientList class for both order and recipe, should we move it into the shared package instead?